mappings/net/minecraft/util/registry/RegistryKey.mapping

```diff
@@ -45,3 +45,7 @@ CLASS net/minecraft/class_5321 net/minecraft/util/registry/RegistryKey
 	METHOD method_29181 of (Lnet/minecraft/class_2960;Lnet/minecraft/class_2960;)Lnet/minecraft/class_5321;
 		ARG 0 registry
 		ARG 1 value
+	METHOD method_31163 isOf (Lnet/minecraft/class_5321;)Z
+		COMMENT Returns whether this registry key belongs to the given registry (according to its type, not whether the registry actually contains this key).
+		ARG 1 registry
```

Comment:
```suggestion
		ARG 1 key
```

Reply:
No, this parameter is specifically the key of the registry this key is supposed to be in, not to be confused with being equal to another key in the same registry.

Reply:
Hum okay